edx-olive/themes
# Adds 404 page which uses tech support email instead

## Description

This pull request updates the support email to the new campus gov email. 

Because the LMS 404 page email uses the `email_from_address` attribute in the Site Configuration in the django admin, we override the 404 page to use `TECH_SUPPORT_EMAIL` because we can't change the `email_from_address` setting.

## Supporting information

* **Jira Tickets**: SE-3703, CAM2-487
* **Related Pull Requests**: https://github.com/edx-olive/olive-secure/pull/189

## Testing instructions

* Activate the changes on Stage
* Reset your password and make sure the "contact technical support email" is `campus@mse.gov.il`
* Open a page that doesn't exist such as `/dashboard/` (include the `/` after) and make sure the email is `campus@mse.gov.il`
